sdk/nodejs/tests_with_mocks/mocks.spec.ts

```diff
@@ -37,12 +38,12 @@ pulumi.runtime.setMocks({
                     publicDns: "ec2-203-0-113-12.compute-1.amazonaws.com",
                     publicIP: "203.0.113.12",
                 };
-                return { id: "i-1234567890abcdef0", state: { ...inputs, ...state } };
+                return { id: "i-1234567890abcdef0", state: { ...args.inputs, ...state } };
             case "pkg:index:MyCustom":
-                assert.strictEqual(custom, true);
-                return { id: name + "_id", state: inputs };
+                assert.strictEqual(args.custom, true);
+                return { id: name + "_id", state: args.inputs };
```

Comment:
This should fix the promise leak
```suggestion
                return { id: args.name + "_id", state: args.inputs };
```

Reply:
🥇